cypress.json

```diff
@@ -4,6 +4,8 @@
     "screenshotOnRunFailure": false,
     "responseTimeout": 20000,
     "reporter": "cypress-multi-reporters",
+    "chromeWebSecurity":false,
```

Comment:
Watch the consistent spacing and styling 😅  Maybe setup the linter and formatter to help you with this file as well.